migrations/20200818171633-users.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    const transaction = await queryInterface.sequelize.transaction()
+
+    try {
+      queryInterface.createTable(
+        'users',
+        {
+          id:      { type: Sequelize.INTEGER, primaryKey: true, autoIncrement: true },
+          email:   { type: Sequelize.STRING},
+          password:   { type: Sequelize.STRING},
+          first_name:   { type: Sequelize.STRING},
+          last_name:   { type: Sequelize.STRING}
+        },
```

Comment:
Essa intendação está meio estranha

```suggestion
          id: {
            type: Sequelize.INTEGER,
            primaryKey: true,
            autoIncrement: true
          },
          email: { type: Sequelize.STRING },
          password: { type: Sequelize.STRING },
          first_name: { type: Sequelize.STRING },
          last_name: { type: Sequelize.STRING }
        },
```

Reply:
Nenhum desses campos deve ser não nulo?

Reply:
isso nenhum deles pode ser nulo,vo ajustar